src/morale.cpp

```diff
@@ -0,0 +1,143 @@
+#include "morale.h"
+#include "itype.h"
+#include "cata_utility.h"
+#include "debug.h"
+#include "output.h"
+
+#define ITEM_NAME_PLACEHOLDER "%i"  // Used to address an item name
```

Comment:
Why not a `const std::string`? Ideally inside the anonymous namespace below.

Also: the new source file needs to be added to src/CMakeLists.txt.